server/src/main/java/keywhiz/KeywhizConfig.java

```diff
@@ -176,6 +183,15 @@ public String getDerivationProviderClass() {
     return backupExportKeys.get(name);
   }
 
+  public RowHmacCheck getRowHmacCheck() {
+    if (rowHmacCheck.equals("enabled")) {
+      return RowHmacCheck.ENABLED;
+    } else if (rowHmacCheck.equals("logging")) {
+      return RowHmacCheck.LOGGING;
+    }
```

Comment:
Having a typo, such as "enbled", result in the check being disabled is bad.

Reply:
Whats the alternative? Defaulting to enforced could also be problematic

Reply:
Throw an exception. This code runs at app startup time and the app won't come up if this field is not properly configured.